### PR TITLE
Bubble and trouble: release notes for bubbling observers

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Fixes https://github.com/bevyengine/bevy-website/issues/1657.

### CODE REVIEW

File: release-content/0.15/release-notes/13991_Minimal_Bubbling_Observers.md [hunk 1]
```diff
@@ -1,4 +1,38 @@
...
+```rust
+impl <E> Event for Pointer<E>{
+    type Traversal = &Parent;
+    const AUTO_PROPAGATE: bool = true;
+}
+```
+
+This means that, unless you call [`Trigger::propagate(false)`], pointer events will be replicated up the hierarchy (accessing the `Entity` stored in the [`Parent`] component) until it reaches the entity root.
+
+Any type that implements the [`Traversal`] trait can be used as the associated type, and can access arbitrary read-only query data from the world.
```
Comment (BenjaminBrienen) [inline release-content/0.15/release-notes/13991_Minimal_Bubbling_Observers.md:28]:
```suggestion
Any type that implements the [`Traversal`] trait can be used as the associated type and can access arbitrary read-only query data from the world.
```

"and" is a coordinating conjunction here

**Review by BenjaminBrienen — CHANGES_REQUESTED** [review]
Really good!

**Review by BenjaminBrienen — APPROVED** [review]

**Review by aevyrie — APPROVED** [review]
I think this is missing credit for @NthTensor?.I seem to recall he drove most of this work, though I might be misremembering.

**Comment by alice-i-cecile** [comment]
Nth is listed as the author here, which is the primary credit :D

**Pull request merged**